src/index.css

```
@@ -7,18 +7,14 @@ html {
   scroll-behavior: smooth;
 }
 
-/* Ensure all sections have smooth transitions */
-* {
-  transition: all 0.3s ease;
-}
-
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)